mentions-generator/src/main/java/de/brandwatch/minianalytics/mentiongenerator/kafka/config/ConsumerConfig.java

```diff
@@ -39,19 +43,22 @@
     }
 
     @Bean
-    public ConsumerFactory<String, String> consumerFactory(){
-        return new DefaultKafkaConsumerFactory<String, String>(consumerConfigs());
+    public ConsumerFactory<String, Object> consumerFactory(){
```

Comment:
Shouldn't this be `<String, Resource>`? Also below in the method.

Reply:
Yes